src/components/Calculator.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable react/button-has-type */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+export default class MyCalculator extends React.Component {
```

Comment:
- [ ] Please use ES6 syntax here in this project.  You have learned how to write ES6 code in JavaScript module.